node/node_genesis.go

```diff
@@ -102,6 +102,7 @@ func (node *Node) SetupGenesisBlock(db ethdb.Database, shardID uint32, myShardSt
 		GasLimit:       params.GenesisGasLimit * 1000,
 		ShardStateHash: myShardState.Hash(),
 		ShardState:     myShardState.DeepCopy(),
+		ExtraData:      []byte("Harmony for One and All, Open Consensus for 10B"),
 	}
```

Comment:
There is a full stop not a comma, in the message. See https://github.com/harmony-one/harmony/pull/1154

Reply:
Good catch .Changed to period.